src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        greeting();
+        Task[] taskStorage = new Task[100];
+
+        boolean is_exit = false;
+        while (!is_exit) {
```

Comment:
Can consider making this main function into a switch-case function to make it more intuitive.